website/feedback/css/feedback.css

```
@@ -652,45 +392,45 @@ button.feedback-btn.small, input[type="submit"].feedback-btn.small {
     justify-content: center;
 }
-.feedbackTab p:before {
-    background-image: url(../image/envelope.jpg);
+
+#feedback-tab p:before {
+    font-family: "Font Awesome 5 Free";
+    font-weight: 900;
+    content: "\f0e0";
+    padding-right: 8px;
+
     opacity: 1.5; /* OPUS - Added to agree with text opacity */
```

Stylelint (17.12.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)